core/src/libraries/Positions.sol

```diff
@@ -29,7 +29,6 @@ function zip(int24[6] memory positions) pure returns (uint144 zipped) {
  * positions like [-100, 100, -100, 100].
  * @param zipped Encoded Uniswap positions. Equivalent to the layout of `int24[6] storage yourPositions`
  * @return positionsOfNonZeroWidth Flattened array of Uniswap positions that may or may not hold liquidity
- * TODO: consider simply checking whether ticks are strictly increasing. if they are, they must be unique and we can save gas elsewhere
```

Comment:
Unrelated to the deposit stuff, but I considered this and realized it was dumb. If ticks were strictly increasing, it'd be impossible to have a narrow position (like [100, 200]) inside of a wide position (like [-5678, 1234]). So the current logic is good.